deploy/operator/parts/crd.yaml

```diff
@@ -462,6 +462,11 @@ spec:
                                 description: "TCP port which used to connect to Zookeeper node"
                                 minimum: 0
                                 maximum: 65535
+                              secure:
+                                type: integer
```

Comment:
- Looks like attempt to make bool out of int type. Probably it would be more proper to use StringBool type in here
- crd.yaml is a generated file and not supposed to be edited manually